lib/deppack.js

```diff
@@ -183,9 +191,13 @@ const xBrowserResolve = (mod, opts, cb) => {
         err = isGlob ? `Could not load global module '${mod}'.` : `Could not load module '${mod}' from '${src}'.`;
 
         if (packages[topLevel] == null) {
-          err = err += ` Possible solution: add '${topLevel}' to package.json and \`npm install\`.`;
+          err += ` Possible solution: add '${topLevel}' to package.json and \`npm install\`.`;
         } else {
-          err = err += ` Possible solution: run \`npm install\`.`;
+          if (overrides[mod]) {
+            err += ` Possible solution: check your overrides in package.json.`;
```

Comment:
I'm not sure I'm following. Suppose I have react in my package.json and an override section for it **but** I forgot to `npm install`. This PR will print the unhelpful error message about checking my overrides